src/filler/handlers/blends/processors/superblends.ts

```diff
@@ -175,6 +177,20 @@ export function superBlendsProcessor(core: CollectionsListHandler, processor: Da
         BlendsUpdatePriority.TABLE_FEATURES.valueOf()
     ));
 
+    destructors.push(processor.onActionTrace(
```

Comment:
It’s not necessary to create this one. The code is updating the blend when there is an update in the row.